src/test/java/hudson/plugins/copyartifact/TriggeredBuildSelectorTest.java

```diff
@@ -926,4 +924,16 @@ public void testMavenModule() throws Exception {
                 b.getWorkspace().child("org.jvnet.hudson.main.test.multimod/moduleB/1.0-SNAPSHOT/moduleB-1.0-SNAPSHOT.jar").exists()
         );
     }
+
+    /**
+     * Creates an empty Maven project with an unique name.
+     *
+     * @return an empty Maven project with an unique name.
+     */
+    private MavenModuleSet createMavenProject() throws IOException {
+        MavenModuleSet mavenModuleSet = j.jenkins.createProject(MavenModuleSet.class, "test"+j.jenkins.getItems().size());
+        mavenModuleSet.setRunHeadless(true);
```

Comment:
Silly to be repeating this incantation in every test using `maven-plugin`. Better to just fix that plugin to set `runHeadless` to true by default when `Main.isUnitTest()`.